packages/python-skia-pathops/build.sh

```diff
@@ -0,0 +1,32 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/fonttools/skia-pathops
+TERMUX_PKG_DESCRIPTION="Python bindings for the Skia library's Path Ops"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="Nguyen Khanh @nguynkhn"
+TERMUX_PKG_VERSION="0.8.0.post1"
+TERMUX_PKG_SRCURL=git+$TERMUX_PKG_HOMEPAGE
+TERMUX_PKG_DEPENDS="libc++, python"
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_PYTHON_COMMON_DEPS="setuptools, wheel, setuptools_scm, 'Cython>=0.28.4'"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	termux_setup_gn
+
+	local _arch=$TERMUX_ARCH
```

Comment:
You probably wanted a `case` here.
```suggestion
	local _arch
	case "$TERMUX_ARCH" in
		'aarch64') _arch='arm64';;
		'arm')     _arch='arm';;
		'x86_64')  _arch='x64';;
		'i686')    _arch='x86';;
		*) termux_error_exit "Architecture not supported by build system"
	esac
```